CRM/Import/Parser.php

```diff
@@ -1128,7 +1128,7 @@ private function _civicrm_api3_deprecated_add_formatted_param(&$values, &$params
 
     // get the formatted location blocks into params - w/ 3.0 format, CRM-4605
     if (!empty($values['location_type_id'])) {
-      static $fields = NULL;
+      $fields = NULL;
       if ($fields == NULL) {
         $fields = [];
       }
```

Comment:
All 4 of these lines duplicate the start of the function and I think all 4 should be removed.
If you don't remove them all and only change L1131 then it changes the meaning.

Reply:
Updated as per your suggestion and recommitted the file.